1.0.4-runtime/init_container.sh

```diff
@@ -4,10 +4,16 @@ mkdir -p /home/LogFiles
 touch /home/LogFiles/dotnet_$WEBSITE_ROLE_INSTANCE_ID_out.log
 echo "$(date) Container started" >> /home/LogFiles/node_$WEBSITE_ROLE_INSTANCE_ID_out.log
 
-if [ $# -eq 0 ]
-  then
-    cd /defaulthome/hostingstart
-    exec dotnet hostingstart.dll
-  else
-    exec "$@"
+[ $# -ne 0 ] && exec "$@"
+
+CSPROJ=`ls -1 /home/site/repository/*.csproj 2>/dev/null | head -1`
```

Comment:
This is cool! Please sprinkle a short comment or two in here as well explaining what this does.